rpcs3/Emu/Cell/Modules/cellSysutil.h

```diff
@@ -79,6 +79,8 @@ enum
 	CELL_SYSUTIL_SYSCHAT_STOP                    = 0x0162,
 	CELL_SYSUTIL_SYSCHAT_VOICE_STREAMING_RESUMED = 0x0163,
 	CELL_SYSUTIL_SYSCHAT_VOICE_STREAMING_PAUSED  = 0x0164,
+
+	CELL_SYSUTIL_USERID_CURRENT = 0,
```

Comment:
This repeats definition in another file. Just use zeros everywhere, it doesn't matter.